index.html

```diff
@@ -36,29 +37,32 @@ <h3 class="connect"><a href="">LET’S CONNECT</a></h3>
 	</section>
 
 	<section class="work">
-		<div>
-			<img src="/img/tonic1.svg" alt="">
-			<h2>TONIC</h2>
-			<ul class="work-exp">
-				<li>Canopy</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>Back End Dev</li>
-				<li><img src="/img/circle.svg" alt=""></li>
-				<li>2015</li>
-			</ul>
-			<p>A daily selection of privately personalized reads; 
-				no accounts or sign-ups required.</p>
-      <ul class="lang">
-				<li><a href="">html</a></li>
-				<li><a href="">css</a></li>
-				<li><a href="">javascript</a></li>
-			</ul>
+		<div id="work_1">
+			<div class="img"><img src="/img/tonic1.svg" alt=""></div>
+			<div class="content_work">
+				<h2>TONIC</h2>
+				<ul class="work-exp">
+					<li>Canopy</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>Back End Dev</li>
+					<li><img src="/img/circle.svg" alt=""></li>
+					<li>2015</li>
+				</ul>
+				<p>A daily selection of privately personalized reads; 
+					no accounts or sign-ups required.</p>
+				<ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">css</a></li>
+					<li><a href="">javascript</a></li>
+				</ul>
```

Comment:
- [ ] While you did an amazing job with the layout of your cards :+1:, I think it would be better if you change some things about it:
  - Kindly don't forget to use the same margin specified in the Figma template for all the elements inside the card :+1: 
  - As the project requires the project description text should be smaller than the row with info ("CANOPY ..") :+1: 

**Figma template**

![Screenshot from 2023-01-19 20-39-46](https://user-images.githubusercontent.com/105117832/213601844-b70a9ae2-3cb6-4f3a-aa54-17fbceca627c.png)

**Your design**

![Screenshot from 2023-01-19 20-40-15](https://user-images.githubusercontent.com/105117832/213601876-3659a40f-a4cd-4315-9014-b2b1338d5f89.png)